frontend/src/config/i18n/en/guidedTour.json

```diff
@@ -0,0 +1,28 @@
+{
+    "next": "Next",
+    "finish": "Finish",
+    "back": "Back",
+    "mapEditor": {
+        "welcome_title": "Welcome to the Map Editor",
+        "toolboxIntro_title": "Toolbox",
+        "layersIntro_title": "Layers",
+        "timelineIntro_title": "Timeline",
+        "baseLayer_title": "Base Layer",

```

Comment:
Also now that I think of it, these strings can already be found in another file afaik.
We should probably merge them into a separate json document to avoid duplicates.

Reply:
"Layers", "Base" and "Plants" can be found in `layers.json`, that is correct. The reason I did not use those values and instead defined them again was for SoC reasons. `layers.json` specifies the translated strings for the layer names, `guidedTour.json` the titles for the Map Editor Tour. While they are fairly similar now, it could still be the case in the future that we want to rename those titles to something different.